tests/testprogs/watchpoint.c

```diff
@@ -17,5 +17,9 @@ int main() {
     return 1;
   }
 
-  *((volatile uint8_t*)addr) = 2;
+
+  uint8_t i = 0;
+  while (1) {
```

Comment:
What does the loop do? Does the `exit()` fix the issue?

Reply:
I couldn't reproduce the issue locally, so I tried a few things at the same time. My theory is that the child was exiting too fast, causing bpftrace to exit fast as well skipping the print. The loop ensures we have a constant rate of events happening, and the `exit()` ensures that bpftrace stop tracing and quit after the first event is captured.

Reply:
Sure, that makes sense. I guess I was worried that the kernel might "lose" an event, but even if it did it's not like we should be testing the kernel interface here.